.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,6 +14,4 @@
 <!-- If this is a fix or otherwise resolves an issue, reference it here with "closes #(issue)" -->
 
 ## Final checklist:
-- [ ] My PR is the minimum possible work for the desired functionality
-- [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have added [alt text](https://webaim.org/techniques/alttext/) to new images included in this PR
```

Comment:
Don't feel strongly here either but would like to remove the checklist as I think doc only PRs would be most common and I never know what I am mean to do in that case. 

Maybe a minor re-word also here to make these more relevant E.g., comment code so others know what it achieves

Could we also add a 'if the PR relates to tooling/infrastructure changes:' before the items - because when I make a doc PR the first 2 items seem a bit awkward/confusing to me?